bindings/nodejs/lib/gpio.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
take the same approach as we do with `build/stage` or `build/soletta_sysroot`. Then you always copy it and use from there, then it will look like installed version.

Sounds reasonable?